src/sqlfmt/mode.py

```diff
@@ -13,7 +13,7 @@ class Mode:
     report config. For more info on each option, see cli.py
     """
 
-    SQL_EXTENSIONS: List[str] = field(default_factory=lambda: [".sql", ".sql.jinja"])
+    SQL_EXTENSIONS: tuple = (".sql", ".sql.jinja")
```

Comment:
This is the newer syntax, but I'd rather have a parameterized generic, and until we can drop support for pythons < 3.9, we need to use `Tuple`, not `tuple`. You'll need to import Tuple from typing on line 3, above (`from typing import List, Tuple`).

```suggestion
    SQL_EXTENSIONS: Tuple[str] = (".sql", ".sql.jinja")
```

Reply:
Syntax was slightly different, but it is changed to `Tuple = (".sql", ".sql.jinja")` now.